frontend/public/components/factory/ListPage/ListPageCreate.tsx

```diff
@@ -66,32 +67,41 @@ export const ListPageCreateDropdown: React.FC<ListPageCreateDropdownProps> = ({
   onClick,
 }) => {
   const [isOpen, setOpen] = React.useState(false);
+
+  const ListCreateDropdownItems = Object.keys(items).map((key) => {
```

Comment:
Since this is not a component...
```suggestion
  const listCreateDropdownItems = Object.keys(items).map((key) => {
```